os/mm/mm_heap/mm_heapinfo_parse_heap.c

```diff
@@ -182,6 +198,24 @@ void heapinfo_parse_heap(FAR struct mm_heap_s *heap, int mode, pid_t pid)
 					heap_dbg("0x%x | %8d |   %c    |            |       |\n", node, node->size, 'F');
 				}
 			}
+#ifdef CONFIG_WATCHDOG
+			if (wd_fd > 0) {
```

Comment:
Line 121 checks wd_fd so that this line looks unnecessary. It could be a defect with always true on the SVACE.